scene/gui/code_edit.h

```diff
@@ -36,6 +36,22 @@
 class CodeEdit : public TextEdit {
 	GDCLASS(CodeEdit, TextEdit)
 
+public:
+	/* Keep enum in sync with:                                           */
+	/* /core/object/script_language.h - ScriptCodeCompletionOption::Kind */
```

Comment:
Can't you just use the other enum, instead of introducing another one that needs to be the same?

Reply:
That's possible, though it felt incorrect to bind a enum from `/core` to `CodeEdit`, not sure of any cases where `enums` aren't local to the class. Also can't bind it to `ScriptCodeCompletionOption` as it's a struct, because of that, if we did cross reference, the docs get confused tying to look for `ScriptCodeCompletionOption`.

Reply:
Ah, it's required for binding. Not sure then, either we move it to CodeEdit or maybe it's fine to have duplicates if they are commented properly.

Reply:
Yeah that crossed my mind too, but as it's used in `script_language`, moving it would make `/core` dependent on `/scene`, which is a no go. 

Not sure there is a clean way out of it as is, without duplicating. The alternative is to not use `enums`, or we rewrite `ScriptCodeCompletionOption` into a bindable class, which is a lot easier said then done.